### ISSUE TITLE
Flaps Axis not working/supported (?)

### ISSUE THREAD

**Issue by eaglerainbow** [it0uchpods/IDG-A32X#85]
### Expected Behavior
Axis control changes on a joystick attached, which are configured to move the flaps, should lead to flaps getting up/down. 

### Current Behavior
Even though configured in the joystick configuration, flaps do not react on changes of the axis.

### Possible Solution
n.a.

### Steps to Reproduce
1. Start simulator with A320, staying on some runway.
2. Make sure an axis of your joystick is mapped to the "flaps" command
3. Make sure that you are unpaused.
4. Move the axis of your joystick.
5. Observe that the flaps don't move.

**Comment by it0uchpods** [comment]
OK, so I need to rewrite the entire flap handling to do this.
I'm working on a new thing called "INTEGRATED SYSTEMS" that is a total rewrite and debut on the MD-11, so this bug will be fixed there first. I'll close this once the fix is done, it will just be a matter of transferring it into this aircraft also.

Kind Regards,
Josh

**Comment by eaglerainbow** [comment]
Please don't hurry. I will be able to go on living without it 😀 It's just a nice to have.
However, if you want to have a beta tester, please get in touch with me (I might need to get some basic "how to get the MD11 powered up" guide, though).

**Comment by it0uchpods** [comment]
:)
I think I know how to make this work. At the time I made the flap handler originally, I did not know you could change the JSBsim source of the flaps input, so it has to overwrite FG's way. I think instead we can do this:

Keys or your axis change /controls/flight/flaps. My system can round to the nearest "detent" and set the lever there. If using keys, this will always match them, but in axis, this will make it work.

Will probably have that on the MD-11 first, if you need the powerup guide, this one is for the PMDG but will work for the "DetailedSystems" and master versions.
https://www.youtube.com/watch?v=sMJ309DZwOs

Kind Regards,
Josh

**Comment by it0uchpods** [comment]
@eaglerainbow would you care to join us on Discord, I am working on this now.
http://discord.gg/UebJyRS

Kind Regards,
Josh

**Comment by eaglerainbow** [comment]
@it0uchpods I general I'd love to!
However, please keep in mind that I am in the CEST time zone (UTC+2 right now). In fact, I am just about to head to work by now...

**Comment by legoboyvdlp** [comment]
No problem - it isn't realtime like IRC or voice chat 😃

**Comment by eaglerainbow** [comment]
Just for the record: With c960558ef21c1a14cef6bbbbadbebd46df95c9dc I have seen and successfully tested a variant of a solution for this bug here on the MD11. 
If we had the equivalent thing here (applied to the flaps configuration of an A32x), then I would be perfectly fine with it (and this issue was resolved).

**Comment by it0uchpods** [comment]
Will be done with integrated systems. 
Because the issue it is solved, and as soon as I port that to the A32X platform it will be fixed and not sooner (because requires significant rewrite) I am closing this for now. Since it is part of the integrated systems platform now it will 100% ported over.
